packages/mobile/src/components/lineup-tile/LineupTileActionButtons.tsx

```diff
@@ -111,24 +122,40 @@ export const LineupTileActionButtons = ({
   const showPremiumAccessStatus = trackId && !doesUserHaveAccess
   const showLeftButtons = !showPremiumAccessStatus && !isUnlisted
 
-  return (
-    <View style={styles.bottomButtons}>
-      <View style={styles.leftButtons}>
-        {showPremiumAccessStatus && premiumConditions != null ? (
+  let content
```

Comment:
super nit, but we're leaving this undefined intentionally in one of our if branches. Might be nice to explicitly set to `null` so the intent is clear?

Reply:
Good call out. I got lazy because typing it was a little annoying, but can update.